doc/changes/devel/13268.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed bug where the file_id is not set when the file contains a directory at the end of the file, by :newcontrib:`Théodore Papadopoulo`.
```

Comment:
Looks like you have contributed a couple of times before :smile: e.g. https://mne.tools/dev/changes/v1.4.html https://mne.tools/dev/changes/v0.19.html
```suggestion
Fixed bug where the file_id is not set when the file contains a directory at the end of the file, by `Théodore Papadopoulo`_.
```